### ISSUE TITLE
Change <title> and desc for b.org homepage

### ISSUE THREAD

**Issue by sutibun** [backdrop-ops/backdropcms.org#149]
As Bd's unofficial marketer, the monkey is **strongly** recommending the following two changes for the homepage. Please @jenlampton no make monkey beg.

These are the simplest and easiest changes to "tickle" Google ^_-). Don't really wanna go into explaining why or how arrived to this but know these aren't done on a whim. *whispers* hours of research

We can always update them later if need be (and prob will) but the faster we can implement them the better for the website.

I have no problem if we hardcode them for the time being (especially given the [metatag module](https://www.drupal.org/project/metatag) hasn't been ported)
## 1) `<title>`

On the homepage, please replace the markup

`<title>Backdrop CMS | Backdrop CMS</title>`

with the following:

`<title>Backdrop CMS: Best FREE Open Source Content Management System</title>`
## 2) `description`

Add the following HTML code on the homepage and make sure it's between `<head>...</head>`

`<meta name="description" content="The top PHP CMS web software to build the fastest & most professional website. Not only is the CMS simple, lightweight & easy to use but is based on the popular secure framework, Drupal">`

if you guys don't feel comfortable mentioning Drupal, change the last part to "...but is based on a popular & secure CMS framework"

**Comment by klonos** [comment]
I see the "monkey business" behind this. Monkey is trying to get popular terms used in google when searching for CMS and software to build a website in general and put all that in the site description. :+1:

**Comment by klonos** [comment]
...perhaps not getting too cocky. Perhaps "One of the best" instead of "The top"?

**Comment by sutibun** [comment]
\*nods\* ^^)b

Ah, didn't think that "best" might imply cockiness or "look" bad. Monkey still prefers what I suggested since

- its optimal for Google in our situation (though not a huge loss if not there)
- nobody really remembers these titles once they clicked over from Google
- if we don't think we're the best, why are we in the fight

Hmm… but if don't feel comfortable you can drop "best" and just have the following:

`<title>Backdrop CMS: FREE Open Source Content Management System</title>`

Also, "free" doesn't need to be all capital. Did it because there are paid CMS out there.

**Comment by jenlampton** [comment]
updated: `<title>Backdrop CMS: Free Open Source Content Management System</title>`

Description tag needs some work... `<meta name="description" content="The top PHP CMS web software to build the fastest & most professional website. Not only is the CMS simple, lightweight & easy to use but is based on the popular secure framework, Drupal">`

**Comment by jenlampton** [comment]
First update made: https://backdropcms.org/ Let's iterate as we feel necessary!

**Comment by jenlampton** [comment]
I'm a little worried about the language `based on` since people might think backdrop is a CMS based on Drupal, a CMF.

**Comment by thejimbirch** [comment]
For reference:

Wordpress:
A semantic personal publishing platform with a focus on aesthetics, web standards, and usability.

Drupal:
Official homepage of the open source content management system. Offers documentation and the source for download and hosts a developers and community ...

Joomla:
Joomla! is the mobile-ready and user-friendly way to build your website. Choose from thousands of features and designs. Joomla! is free and open source.

**Comment by thejimbirch** [comment]
`Backdrop CMS is simple, lightweight, and easy to use content management system used to build creative, professional websites.`

**Comment by jenlampton** [comment]
How about `Backdrop CMS is simple, lightweight, and easy to use. A PHP content management system used to build creative, professional websites.`

**Comment by docwilmot** [comment]
Prefer @thejimbirch version, but missing an "a"; I think the "PHP" is not suited there. One thought though, appeal to aesthetics too? 

`Backdrop CMS is a simple, lightweight, and easy to use content management system used to build beautiful, professional websites.` or maybe "attractive" instead of "beautiful".

**Comment by jenlampton** [comment]
I like `Backdrop CMS is a simple, lightweight, and easy to use content management system used to build attractive, professional websites.`

**Comment by jenlampton** [comment]
Should we say anything about SMB + Non-Profits?

**Comment by thejimbirch** [comment]
Backdrop CMS is a simple, lightweight, and easy to use open source content
management system used to build attractive, professional websites.

**Comment by sutibun** [comment]
Prob went overkill on my first try.

>  I think the "PHP" is not suited there ... or maybe "attractive" instead of "beautiful".

Some of the words I included are there for a purpose. They'll help with the site's SEO efforts. Guess need a better balance. How about the following?

> The simple, lightweight and easy to use content management software in PHP. Join savvy site builders who build creative, professional websites with Backdrop CMS.

I don't think it's necessary to start this with "Backdrop CMS" since the title already has that covered.

**Comment by Dinornis** [comment]
I would go with what monkey suggested in the original post as it is using phrases that people actually use when searching for CMS.  To me it looks like monkey knows its business.
>...perhaps we shouldn't be too cocky. Perhaps "One of the best..." instead of "The top..."?

Can you name a single CMS that you think is better?  I tried a few and can't think of one that would come even close.  In this instance it does not really matter because it is much more likely that someone searching for a CMS will use the phrase "top cms" or "best cms" rather than "one of the best cms".  As monkey said, the intentions are to get higher ranking on the search results page, let monkey do.

**Comment by Dinornis** [comment]
@jenlampton 
> updated: <title>Backdrop CMS: Free Open Source Content Management System</title>

Looks like the home page title is back to <title>Backdrop CMS | Backdrop CMS</title>

**Comment by klonos** [comment]
>>...perhaps we shouldn't be too cocky. Perhaps "One of the best..." instead of "The top..."?
>
>Can you name a single CMS that you think is better?

I appreciate your enthusiasm @Dinornis. Maybe I'm just getting old and loosing the "passion" that is characteristic of the young(er). It is undeniable that we all have different personalities and perhaps it's just me, but we simply cannot claim that we are *the best* there is. Especially when none of us can have possibly tried *every single* other CMS out there.

During my presentations, I introduce myself as a "CMS enthusiast" and people that have been in the GitHub issue queue have seen me open issues like #1388 (and any sub-issue that come with it). So, short answer is that no, we are not *the best*. 

Speaking of presentations, during a recent one where there was also talk about another CMS, I was impressed by certain aspects of their UI (I even filed an issue here to consider having our respective UI work in a similar fashion). Nevertheless, the (admittedly very much younger than me) presenter claimed that they are "the best" and in fact that they are the "most secure" CMS out there. Perhaps the intention was to use "one of the best" and "one of the most secure" there, but being young(er), I guess the passion might have taken over for a moment there. If not, then I am sorry but I will have to make some terrible assumptions about the both the person and the CMS they were presenting.

You see, **I think that modesty is in order at all times**. I'd rather we let others talk about our qualities and say that "we rock" instead of proclaiming ourselves the king of the world. Doing so would only show signs of immaturity and arrogance and I think that it would make people considering adopting our project think twice. That's my personal opinion at least.

> "best cms" rather than "one of the best cms"

 "one of the best cms" includes "best cms". I think that we should be fine with robots indexing that 😉

**Comment by Dinornis** [comment]
I agree

**Comment by klonos** [comment]
> Looks like the home page title is back to <title>Backdrop CMS | Backdrop CMS</title>

Yes, I think that this was fixed but perhaps a recent update might have reverted things to defaults. You see, every Backdrop installation appends `| %site_name%` at the end of the page `<title>` by default (and the name of the site is "Backdrop CMS"). The unfortunate duplication occurs because the home page happens to be a page with title "Backdrop CMS" as well.

I think that in Backdrop we need to decouple the page title (usually a `<h1>`) and the `<title>` of pages. It is not as easy a task as it may seem because we have inherited code from D7 that is tightly integrated with other subsystems, but we have quite a few issues filed for related problems. Examples include #1508, #1456, #1790 and others. We'll eventually get there 😄 

Reopening this issue in order to fix the specific issue with b.org home page.
